Themes/default/languages/Post.english.php

```diff
@@ -43,6 +43,8 @@
 $txt['reset_votes_check'] = 'Check this if you want to reset all vote counts to 0.';
 $txt['votes'] = 'votes';
 $txt['attach'] = 'Attach';
+$txt['attachment'] = 'Attachment';
+$txt['attachments'] = 'Attachments';
```

Comment:
Same as $txt['permissiongroup_attachment'], ManagePermissions.english.php L289

Reply:
Not an issue. :)

Reply:
Thanks, wasn't sure how those particular cases were handled.